### ISSUE TITLE
IMDB CSV structure changed

### ISSUE THREAD

**Issue by paulgendek** [RileyXX/IMDB-Trakt-Syncer#93]
### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Bug Description

The columns and `Title Type` values have changed in the IMDB exports.

Watchlist:
```
Position,Const,Created,Modified,Description,Title,Original Title,URL,Title Type,IMDb Rating,Runtime (mins),Year,Genres,Num Votes,Release Date,Directors,Your Rating,Date Rated
```

Ratings:
```
Const,Your Rating,Date Rated,Title,Original Title,URL,Title Type,IMDb Rating,Runtime (mins),Year,Genres,Num Votes,Release Date,Directors
```

I made the following changes in `imdbData.py` and everything is working correctly again:

Watchlist:
```python
                    title = row[5]
                    year = row[11]
                    imdb_id = row[1]
                    date_added = row[2]
                    # Convert date format
                    date_added = datetime.strptime(date_added, '%Y-%m-%d').strftime('%Y-%m-%dT%H:%M:%S.000Z')
                    if "TV Series" in row[8] or "TV Mini Series" in row[8]:
                        media_type = "show"
                    elif "TV Episode" in row[8]:
                        media_type = "episode"
                    elif "Movie" in row[8] or "TV Special" in row[8] or "TV Movie" in row[8] or "TV Short" in row[8] or "Video" in row[8]:
                        media_type = "movie"
```

Ratings:
```python
                    title = row[3]
                    year = row[9]
                    rating = row[1]
                    imdb_id = row[0]
                    date_added = row[2]
                    # Convert date format
                    date_added = datetime.strptime(date_added, '%Y-%m-%d').strftime('%Y-%m-%dT%H:%M:%S.000Z')
                    if "TV Series" in row[6] or "TV Mini Series" in row[6]:
                        media_type = "show"
                    elif "TV Episode" in row[6]:
                        media_type = "episode"
                    elif "Movie" in row[6] or "TV Special" in row[6] or "TV Movie" in row[6] or "TV Short" in row[6] or "Video" in row[6]:
                        media_type = "movie"
```

### Environment

macOS using default installation

**Comment by RileyXX** [comment]
Thank you very much for reporting this! And nice job for figuring out what the problem was and implementing a fix. 

I have addressed this issue in the latest patch ([v1.9.1](https://github.com/RileyXX/IMDB-Trakt-Syncer/releases/tag/v1.9.1)). I made a few changes which should help prevent this from happening again, and added some checks to prevent the script from running and raise a specific error should the CSV data structure change again in the future.
